cinema/models.py

```diff
@@ -90,9 +88,7 @@
     movie_session = models.ForeignKey(
         MovieSession, on_delete=models.CASCADE, related_name="tickets"
     )
-    order = models.ForeignKey(
-        Order, on_delete=models.CASCADE, related_name="tickets"
-    )
+    order = models.ForeignKey(Order, on_delete=models.CASCADE, related_name="tickets")
```

Comment:
Verify that the `Order` model is correctly defined and migrated, as it is referenced here as a foreign key.